project/stac_api/models.py

```diff
@@ -1,28 +1,307 @@
 from django.contrib.gis.db import models
+from django.utils.translation import gettext_lazy as _
+from django.core.exceptions import ValidationError
+from django.contrib.postgres.fields import ArrayField
+import re
 
 # pylint: disable=fixme
 # TODO remove this pylint disable once this is done
 
 # st_geometry bbox ch
 BBOX_CH = 'POLYGON((2317000 913000,3057000 913000,3057000 1413000,2317000 1413000,2317000 913000))'
 
+# I allowed myself to make excessive use of comments below, as this is still work in progress.
+# all the comments can be deleted later on
```

Comment:
NOPE! I like excessive comments!! We should have more of them in general. It makes collaboration it much easier, specially when you look at older code